### ISSUE TITLE
Inline formatting of text using * or _

### ISSUE THREAD

**Issue by RCMainak** [kvirc/KVIrc#1893]
Hello everyone! This is not my strength to write C++ code or whatever you guyz write. I think it would be nice if the text inside the astricks( \* ) sign turns into **BOLD** and the text enclosed by underscore( _ ) turns into _italic_ . I think it's not hard to do just needs someone to care enough.

**Comment by staticfox** [comment]
Basically how Skype works?

**Comment by Dessa** [comment]
i think irssi does at least the former as well.

**Comment by wodim** [comment]
Irssi does it with \* too. It's something to think about.

**Comment by extioStorm** [comment]
a script can solve something this simple? talking about the text sent to the chat room?

**Comment by un1versal** [comment]
GitHub mardown does

`*this is italic*` _this is italic_
`**this is bold**` **this is bold**
`***this is bold italic***` **_this is bold italic**_

We already have bold with other ctrl characters, what we dont have is italic

**Comment by un1versal** [comment]
correction we now have italic

**Comment by un1versal** [comment]
Do we really want to implement this? Why is this better than the control characters we have now?

**Comment by Heufneutje** [comment]
I wouldn't mind seeing it as an alternative to control characters ~~clientside~~ during input and just have KVIrc translate them to control characters upon sending.

**Comment by un1versal** [comment]
yes sure, but its not any easier to get than a ctrl char as you still need to press the same amount of keys to get either `*` or `_`

and this request only accounts for Bold, what about _italic_, underline, what about **_italic bold**_ or italic underline (github cant underline) what about for italic bold and underline? how many of these shall we use?

clearly not a skype user...
